single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultLabelProvider.java

```diff
@@ -179,25 +163,22 @@ public List<ElectricalNodeInfo> getElectricalNodesInfos(VoltageLevelGraph graph)
                 .collect(Collectors.toList());
     }
 
-    private void addBranchStatusDecorator(List<NodeDecorator> nodeDecorators, Node node, Direction direction, Connectable<?> c) {
-        OperatingStatus<?> branchStatus = (OperatingStatus<?>) c.getExtension(OperatingStatus.class);
-        if (branchStatus != null) {
-            switch (branchStatus.getStatus()) {
-                case PLANNED_OUTAGE:
-                    nodeDecorators.add(getBranchStatusDecorator(node, direction, PLANNED_OUTAGE_BRANCH_NODE_DECORATOR));
-                    break;
-                case FORCED_OUTAGE:
-                    nodeDecorators.add(getBranchStatusDecorator(node, direction, FORCED_OUTAGE_BRANCH_NODE_DECORATOR));
-                    break;
-                default:
-                    break;
+    private void addBranchStatusDecorator(List<NodeDecorator> nodeDecorators, Node node, Direction direction, Identifiable<?> identifiable) {
+        if (identifiable != null) {
+            OperatingStatus<?> branchStatus = (OperatingStatus<?>) identifiable.getExtension(OperatingStatus.class);
+            if (branchStatus != null) {
+                switch (branchStatus.getStatus()) {
+                    case PLANNED_OUTAGE -> nodeDecorators.add(getBranchStatusDecorator(node, direction, PLANNED_OUTAGE_BRANCH_NODE_DECORATOR));
+                    case FORCED_OUTAGE -> nodeDecorators.add(getBranchStatusDecorator(node, direction, FORCED_OUTAGE_BRANCH_NODE_DECORATOR));
+                    default -> { }
```

Comment:
Same thing, you need to add a comment in order not to have a code smell, maybe with the explicit `IN_OPERATION` case